### DISCUSSION TITLE
Allow `v fmt` to format `v.mod` as well

### DISCUSSION THREAD

**Discussion by x80486** [vlang/v#18622] [Ideas]
### Describe the feature

`v.mod` is a file specific to `V`. Since `fmt` is actually the command used to format `V` source code, it could be extended to format `v.mod` as well.

### Use Case

Because `v.mod` is a file specific to `V`, there is no need to rely on manual formatting for the same.

### Proposed Solution

N/A

### Other Information

N/A

### Acknowledgements

- [ ] I may be able to implement this feature request
- [ ] This feature might incur a breaking change

### Version used

V 0.3.2 a987f84

### Environment details (OS name and version, etc.)

OS: linux, Linux version 5.15.85-1-lts (linux-lts@archlinux) (gcc (GCC) 12.2.0, GNU ld (GNU Binutils) 2.39.0) #1 SMP Thu, 22 Dec 2022 06:22:00 +0000
Processor: 12 cpus, 64bit, little endian, Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz
CC version: cc (GCC) 12.2.0

getwd: /home/x80486/Workshop/v-sandbox
vmodules: /home/x80486/.vmodules
vroot: /home/x80486/.asdf/installs/v/weekly.2022.51
vexe: /home/x80486/.asdf/installs/v/weekly.2022.51/v
vexe mtime: 2022-12-25 01:19:02
is vroot writable: true
is vmodules writable: true
V full version: V 0.3.2 6a179a2

Git version: git version 2.39.0
Git vroot status: weekly.2022.51 (2 commit(s) behind V master)
.git/config present: true
thirdparty/tcc status: thirdparty-linux-amd64 12f392c3

**Comment by Delta456** [comment]
Do we have a syntax grammar made for `v.mod`?
